ls1intum/Apollon
# `Development`: Update dependencies

### Motivation and Context
This PR updates the dependencies of Apollon.

### Steps for Testing
1. Clone the repo
2. Execute commands: `yarn install`, `yarn build` followed by `yarn test`
3. Observe no tests are failing and the application is running successfully with `yarn start` command
